### PR TITLE
Feature/default search state

### PR DESCRIPTION
**Pull request by ghostPepperWonton**
This removes foundation equalizer for a few issues and uses flex to position and size instead.

There's a huge performance boost, so if we can get this into staging, we can start testing.

### CODE REVIEW

**Review by ben-rabid — APPROVED** [review]

**Review by ben-rabid — APPROVED** [review]

File: app/assets/javascripts/search-result-load-in.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+$(document).ready(function() {
+  let searchResult = 0;
```
[the remark this answers is not in the document]
Comment (elspeth-rabid) [reply app/assets/javascripts/search-result-load-in.js:2]: You are transpiling down to ES5?

**Review by elspeth-rabid — CHANGES_REQUESTED** [review]
It seems your scss lint rules might need a review.
What is the convention you are going for? 
BEMS with underscored elements and dashed modifiers?

I am not aware of babel doing any conversions of ES6 to ES5 in this project. Wouldn't the use of `let` crash IE?

So great to see this page getting a fixer over!

File: app/assets/javascripts/search-result-load-in.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (ghostPepperWonton) [reply app/assets/javascripts/search-result-load-in.js:2]: I just didn't have the ES6 setting on for ESLint

**Comment by ghostPepperWonton** [comment]
@elspeth-rabid Yeah, I'd like to address the SCSS stuff at some point soon. I tried to switch what I was doing over to BEM style stuff, but there's a lot of cruft in there of alternating underscored and dashed CSS classes, many long chains of words that don't give a whole lot of semantic information. I want to make that a separate issue.

I was also using `let` because Code Climate originally complained that I didn't, but I hadn't realized that the ESLint settings weren't including ES6, which they now do.

**Review by elspeth-rabid — APPROVED** [review]
STARS

**Pull request merged**